code/modules/mob/living/basic/basic.dm

```diff
@@ -91,6 +91,11 @@
 	///This damage is taken when atmos doesn't fit all the requirements above. Set to 0 to avoid adding the atmos_requirements element.
 	var/unsuitable_atmos_damage = 1
 
+//MONKESTATION EDIT START
+	/// List of weather immunity traits that are then added on Initialize(), see traits.dm.
+	var/list/weather_immunities
```

Comment:
This var is never actually used, you'll need to copy this code from simple animals to basic mobs:

https://github.com/Monkestation/Monkestation2.0/blob/a870daf03499064f57e357246d9b9b27a8d748d1/code/modules/mob/living/simple_animal/simple_animal.dm#L175-L176

Reply:
fix